upgrade-steps.md

```diff
@@ -70,4 +70,51 @@ This documents the steps taken to upgrade VPB from ruby 1.93/rails 3.2 to the la
 |app/assets/javascripts/application.js|0%|0% (not needed for rails/ruby upgrade)|
 |app/assets/javascripts/aws-upload.js|0%|0% (not needed for rails/ruby upgrade)|
 
+## Dependency Versions
 
+|gem                      |Environment|Latest |Latest Ruby|Initial 3|Final 3  |Audited/Keep|Audit Notes|
+|-------------------------|-----------|-------|-----------|---------|---------|------------|-----------|
+|capistrano               |all        |3.12.1 |>= 2.0     |2.14.1   |*removed*|---         |Already removed |
+|capistrano-maintenance   |all        |1.2.1  |>= 0       |0.0.2    |*removed*|---         |Already removed |
+|sass-rails               |assets     |6.0.0  |>= 0       |3.2.6    |3.2.6    |Y/N         |No sass files present |
+|coffee-rails             |assets     |5.0.0  |>= 0       |3.2.2    |3.2.2    |Y/N         |No coffee files present |
+|therubyracer             |assets     |0.12.3 |>= 0       |0.12.1   |0.12.1   |Y/N?        |git commmit says its for asset precompile |
+|uglifier                 |assets     |4.2.0  |>= 1.9.3   |1.3.0    |1.3.0    |Y/N?        |not referenced in asset precompile |
+|turbo-sprockets-rails3   |assets     |0.3.14 |>= 0       |0.3.6    |0.3.6    |Y/N?        |deployment will change |
+|jquery-ui-rails          |all        |6.0.1  |>= 0       |4.0.0    |4.0.0    |Y/N?        |Not used in asset pipeline that I can see |
+|warden                   |all        |1.2.8  |>= 0       |1.2.1    |1.2.1    |Y/N?        |Not used? |
+|paperclip                |all        |6.1.0  |>= 2.1.0   |3.4.0    |3.4.0    |Y/N?        |Thumbnails now auto generated, need to verify can't upload |
+|nokogiri                 |all        |1.10.9 |>= 2.3.0   |1.5.6    |1.5.6    |Y/N?        |Not used? rspec? |
+|xpath                    |all        |3.2.0  |>= 2.3     |0.1.4    |0.1.4    |Y/N?        |Not used? rspec? |
+|exception_notification   |all        |4.4.0  |>= 2.0     |3.0.0    |3.0.0    |Y/N?        |Partially setup for production |
+|rdoc                     |all        |6.2.1  |>= 2.4.0   |3.12     |3.12.2   |Y/N?        |Was used in rake task, no longer used |
+|tinymce-rails-imageupload|all        |3.5.8.6|NONE       |3.5.6.4  |3.5.6.4  |Y/N?        |Was used by wysihat_files for thumbnails |
+|debugger-ruby_core_source|dev        |1.3.8  |>= 0       |*added*  |1.3.8    |Y/N?        |Installed so debugger gem would install |
+|debugger                 |dev        |1.6.8  |>=0        |1.6.8    |1.6.8    |Y/N?        |Remove to reduce surface area |
```

Comment:
Yeah you can remove this. I it is a pain to get working with updated ruby. It is easy to use `pry`. But I think you won't be able to add that until ruby is updated.